test/integration/scripting_spec.mjs

```diff
@@ -2513,4 +2513,139 @@ describe("Interaction", () => {
+
+          const firstInputValue = await page.$eval(
+            firstInputSelector,
+            el => el.value
+          );
+          expect(firstInputValue)
+            .withContext(`In ${browserName}`)
+            .toEqual("16-Mar-75");
+
+          const secondInputValue = await page.$eval(
+            secondInputSelector,
+            el => el.value
+          );
+          expect(secondInputValue)
+            .withContext(`In ${browserName}`)
+            .toEqual("01:23:45");
+
+          await page.click(firstInputSelector);
+          await waitForSandboxTrip(page);
+
+          const thirdInputValue = await page.$eval(
+            thirdInputSelector,
+            el => el.value
+          );
+          expect(thirdInputValue)
+            .withContext(`In ${browserName}`)
+            .toEqual("3/16/1975 01:23");
+        })
+      );
+    });
+  });
+
+  describe("Date HTML element with initial values", () => {
+    let pages;
+
+    beforeEach(async () => {
+      pages = await loadAndWait("dates_save.pdf", "[data-annotation-id='26R']");
+    });
+
+    afterEach(async () => {
+      await closePages(pages);
+    });
+
+    it("must check that the inputs are correct", async () => {
+      await Promise.all(
+        pages.map(async ([browserName, page]) => {
+          await waitForScripting(page);
+          await waitForSandboxTrip(page);
+
+          const firstInputSelector = "[data-annotation-id='26R'] > input";
+          await page.waitForSelector(`${firstInputSelector}[type="text"]`);
+          await page.click(firstInputSelector);
+          await waitForSandboxTrip(page);
+          await page.waitForSelector(`${firstInputSelector}[type="date"]`);
+          const firstInputValue = await page.$eval(
+            firstInputSelector,
+            el => el.value
+          );
+          expect(firstInputValue)
+            .withContext(`In ${browserName}`)
+            .toEqual("2025-07-01");
+
+          const secondInputSelector = "[data-annotation-id='27R'] > input";
+          await page.waitForSelector(`${secondInputSelector}[type="text"]`);
+          await page.click(secondInputSelector);
+          await waitForSandboxTrip(page);
+          await page.waitForSelector(`${secondInputSelector}[type="time"]`);
+          const secondInputValue = await page.$eval(
+            secondInputSelector,
+            el => el.value
+          );
+          expect(secondInputValue)
+            .withContext(`In ${browserName}`)
+            .toEqual("00:34:56");
+
+          const thirdInputSelector = "[data-annotation-id='28R'] > input";
+          await page.waitForSelector(`${thirdInputSelector}[type="text"]`);
+          await page.click(thirdInputSelector);
+          await waitForSandboxTrip(page);
+          await page.waitForSelector(
+            `${thirdInputSelector}[type="datetime-local"]`
+          );
+          await page.$eval(thirdInputSelector, el => {
+            el.value = "2025-07-02T12:34:00";
```

Comment:
This is setting a value, but after this the test ends so we don't actually check anything here. Shouldn't this be an `expect` statement instead like the ones above to check that the initial value from the PDF is correct?